keras_retinanet/bin/debug.py

```diff
@@ -177,7 +179,13 @@ def run(generator, args):
             image, image_scale = generator.resize_image(image)
             annotations[:, :4] *= image_scale
 
-        anchors = anchors_for_shape(image.shape)
+        # optionally load config parameters
+        anchor_params = None
+        if args.config:
+            config = read_config_file(args.config)
+            anchor_params = parse_anchor_parameters(config)
+
+        anchors = anchors_for_shape(image.shape,anchor_params=anchor_params)
```

Comment:
Please add a space between the two arguments of the function, after the comma, for styling and building purposes.

Reply:
Fixed